File: codewars/python/8kyu/is_it_palindrome.py
def is_palindrome(s: str):
    if s.lower()[::-1] == s.lower():
        return True
    return False


if __name__ == '__main__':
    print(is_palindrome('a'))
    print(is_palindrome('aba'))
    print(is_palindrome('Abba'))
    print(is_palindrome('malam'))
    print(is_palindrome('walter'))
    print(is_palindrome('kodok'))
    print(is_palindrome('Kasue'))

File: codewars/dart/8kyu/is_it_palindrome.dart
void main(List<String> args) {
  print(isPalindrome('a'));
  print(isPalindrome('aba'));
  print(isPalindrome('Abba'));
  print(isPalindrome('hello'));
  print(isPalindrome('Bob'));
  print(isPalindrome('Madam'));
  print(isPalindrome('AbBa'));
  print(isPalindrome(''));
}

bool isPalindrome(String str) {
  final reversedStr =
      String.fromCharCodes(str.toLowerCase().runes.toList().reversed);
  if (reversedStr == str.toLowerCase()) {
    return true;
  }
  return false;
}
